docs/_static/llama-stack-spec.yaml

```diff
@@ -5460,6 +5437,32 @@ components:
         - vector_db_id
         - chunks
       title: InsertChunksRequest
+    ProviderInfo:
+      type: object
+      properties:
+        api:
+          type: string
+        provider_id:
+          type: string
+        provider_type:
+          type: string
+        config:
+          type: object
+          additionalProperties:
+            oneOf:
+              - type: 'null'
+              - type: boolean
+              - type: number
+              - type: string
+              - type: array
+              - type: object
+      additionalProperties: false
+      required:
+        - api
+        - provider_id
+        - provider_type
+        - config
```

Comment:
Is config mandatory, or if empty should it be returned as none rather than an empty dictionary?